apps/web/components/dashboard/widget-settings-dialog.tsx

```
@@ -177,8 +201,77 @@ export function WidgetSettingsDialog() {
+												case "line":
+													Icon = LineChart;
+													break;
+												case "donut":
+													Icon = PieChart;
+													break;
+												case "number":
+													Icon = Binary;
+													break;
+											}
+											return (
+												<div className="border-input shadow-xs has-data-[state=checked]:border-primary/50 has-focus-visible:border-ring has-focus-visible:ring-[3px] has-focus-visible:ring-ring/50 relative flex cursor-pointer flex-col items-center gap-3 rounded-md border px-2 py-6 text-center outline-none transition-[color,box-shadow]">
```

Biome (2.3.13)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)